fastapi/tests/test_fastapi_demo.py

```diff
@@ -17,6 +18,7 @@
 from ..models.fastapi_endpoint_demo import EndpointAppInfo, ExceptionType
 
 
+@odoo.tests.tagged("post_install", "-at_install")
```

Comment:
And I don't understand why this change suddenly requires tests to run post-install.

Reply:
It was by chance...